apps/client/tailwind.config.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
         },
         green: {
           light: 'hsl(var(--light-green))',
+          mediumgreen: 'hsl(var(--medium-green))',
```

Comment:
this should just be called 'medium'

otherwise you're gonna have to write 'text-green-mediumgreen' lol

Reply:
change it

Reply:
k